.github/workflows/docs.yml

```diff
@@ -0,0 +1,59 @@
+name: docs
+
+# Adapted from  https://github.com/mitmproxy/pdoc/blob/316554dd40e69a407800dcf08cea622529b37a7c/.github/workflows/docs.yml
+
+# build the documentation whenever there are new commits on the trunk
+on:
+  push:
+    branches:
+      - trunk
+      - 19/build-api-docs # XXX
```

Comment:
why also this branch?

Reply:
that was for dev. forgot to push https://github.com/informalsystems/apalache-chai/pull/40/commits/f6923f130135e382fa5a6b5abe1ff40f8bc47401

thanks for  the review!